src/main/java/io/cdap/plugin/gcp/bigquery/common/BigQueryBaseConfig.java

```diff
@@ -109,34 +179,4 @@ private Bucket getBucketIfExists(Storage storage, String bucketName) {
     }
     return bucket;
   }
-
-  public void validateCmekKeyLocation(@Nullable CryptoKeyName cmekKeyName, @Nullable String tableName,
```

Comment:
Why do we need to remove this method? Since AbstractBigQueryConfig and BigQuerySQLEngineConfig are duplicating this method. Can we keep it here and remove it from those classes? If we decide to keep it here, please keep the above method `getBucketIfExists` private.

Reply:
Yeah, good catch. I intended to move validateCmekKey since this is different for BigQuerySQLEngineConfig and BigQuerySourceConfig and moved this one instead. I'll fix this in the next rev.